plugins/importexport/crossref/templates/index.tpl

```diff
@@ -11,7 +11,10 @@
 {include file="common/header.tpl" pageTitle="plugins.importexport.crossref.displayName"}
 {/strip}
 
-{if !empty($configurationErrors) || !$currentContext->getSetting('publisherInstitution')|escape || (!$exportArticles && !$exportIssues)}
+{if !empty($configurationErrors) || 
+	!$currentContext->getSetting('publisherInstitution')|escape || 
+	(!$exportArticles && !$exportIssues) || 
+	(!$currentContext->getSetting('onlineIssn')|escape && !$currentContext->getSetting('pringIssn')|escape)}
```

Comment:
1. Should be `printIssn`, I assume? (`t`).

2. You shouldn't need to use `|escape` since the output isn't getting printed. 

3. I like how you've tidied up the layout here. Much more readable.